pika/adapters/gevent_connection.py

```diff
@@ -0,0 +1,64 @@
+"""Use pika with the GEVENT"""
+
+import gevent
+
+from pika.adapters import BaseConnection
+from pika.heartbeat import HeartbeatChecker
+
+
+class PIKAGeventHeartbeatChecker(HeartbeatChecker):
```

Comment:
Please use `Pika` instead of `PIKA`

Reply:
OK, I will fix this and add test in next commit.